chapters/07_chapter_6.tex

```diff
@@ -0,0 +1,390 @@
+\chapter{Introduction to micro-architectural attacks}
+
+
+
+Clémentine Maurice, CNRS, IRISA\\
```

Comment:
Please use \'{e} to avoid problems. Please apply this and the other comments everywhere in the code, I will not mention each place it appears.

Reply:
Done